### PR TITLE
[ADD] French translations

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/i18n/fr.po [hunk 1]
```diff
@@ -6,23 +6,25 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 11.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-10-09 16:25+0000\n"
-"PO-Revision-Date: 2018-10-09 12:29-0400\n"
-"Last-Translator: \n"
+"POT-Creation-Date: 2018-10-10 20:49+0000\n"
+"PO-Revision-Date: 2018-10-10 20:49+0000\n"
+""Last-Translator: <:uis GARCIA RUIZ luis,garcia@savoirfairelinux.com>\n"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @eilst why "" double, and for the translator name <:uis it looks incorrect

File: project_event/i18n/fr.po [hunk 2]
```diff
@@ -570,14 +602,16 @@ msgid "Task Category"
 msgstr "Catégorie de tâches"
 
 #. module: project_event
-#: model:ir.ui.view,arch_db:project_event.view_project_task_search
+#: model:ir.ui.view,arch_db:project_event.project_activity_search
+#: model:ir.ui.view,arch_db:project_event.project_task_search
 msgid "Task Client"
-msgstr "Client de la tâche"
+msgstr "Task Client"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @eilst Client de la tâche

File: project_event/i18n/fr.po [hunk 3]
```diff
@@ -570,14 +602,16 @@ msgid "Task Category"
...
 
 #. module: project_event
-#: model:ir.ui.view,arch_db:project_event.view_project_task_search
+#: model:ir.ui.view,arch_db:project_event.project_activity_search
+#: model:ir.ui.view,arch_db:project_event.project_task_search
 msgid "Task Responsible"
-msgstr "Responsable de la tâche"
+msgstr "Task Responsible"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @eilst Responsable de la tâche

File: project_event/i18n/fr.po [hunk 4]
```diff
@@ -478,8 +496,14 @@ msgstr "Demander"
 msgid "Requested"
 msgstr "Demandé"
 
+#. module: project_event
+#: model:ir.model.fields,field_description:project_event.field_project_task_reservation_event_id
+msgid "Reservation event"
+msgstr "Reservation event"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @eilst translation

File: project_event/i18n/fr.po [hunk 5]
```diff
@@ -452,16 +465,21 @@ msgstr "Option"
...
 #: model:ir.ui.view,arch_db:project_event.project_event_tree
 msgid "Project Event"
 msgstr "Événement de projet"
 
 #. module: project_event
-#: model:ir.ui.view,arch_db:project_event.view_project_task_search
+#: model:ir.ui.view,arch_db:project_event.project_task_search
 msgid "Project Task Search"
-msgstr "Recherche de tâche de projet"
+msgstr "Project Task Search"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: not translated

File: project_event/i18n/fr.po [hunk 6]
```diff
@@ -452,16 +465,21 @@ msgstr "Option"
 msgid "Project"
 msgstr "Projet"
 
+#. module: project_event
+#: model:ir.ui.view,arch_db:project_event.project_activity_search
+msgid "Project Activity Search"
+msgstr "Project Activity Search"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @eilst not transalated

**Review by rimbendhaou — CHANGES_REQUESTED** [review]
@eilst some missing translation

File: project_event/i18n/fr.po (hunk 5 above)
Comment (eilst) [reply project_event/i18n/fr.po]: Done

File: project_event/i18n/fr.po (hunk 6 above)
Comment (eilst) [reply project_event/i18n/fr.po]: Done

**Review by rimbendhaou — APPROVED** [review]
looks ok

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**